installation/sql/mysql/extensions.sql

```diff
@@ -957,21 +956,21 @@ CREATE TABLE IF NOT EXISTS `#__guidedtour_steps` (
-(5, 1, 'URL', '', 1, '<p>Add URL of the page from where you want to start your Tour. </p>', 0, 1, 'bottom', '#jform_url', '', 0, '2021-09-17 18:44:26', 430, '2021-09-17 18:48:25', 430, 1),
-(6, 1, 'overlay', '', 1, '<p>Select if you want an overlay effect or not.</p>', 0, 1, 'bottom', '#jform_overlay1', '', 0, '2021-09-17 18:45:34', 430, '2021-09-17 18:48:36', 430, 1),
-(7, 1, 'save and close', '', 1, '', 0, 1, 'bottom', '#save-group-children-save', '', 0, '2021-09-17 18:46:52', 430, '2021-09-17 18:50:35', 430, 1),
-(8, 1, 'Create steps for your Tour', '', 1, '<p>Click on steps icon in the right</p>', 0, 1, 'right', '.btn-info ', '', 0, '2021-09-17 18:52:03', 430, '2021-09-17 18:53:08', 430, 1),
-(9, 1, 'Click here!', '', 1, '<p>Click here to create a new Step for your Tour</p>', 0, 1, 'bottom', '.button-new', '', 0, '2021-09-17 18:55:27', 430, '2021-09-17 18:55:27', 430, 1),
+(1, 1, 'Click here!', '', 1, '<p>This Tour will show you that how you can create a new Guided Tour in Joomla! Backend</p>', 0, 1, 'bottom', '.button-new', '', '2021-09-17 18:35:44', 430, '2021-09-17 18:35:44', 430, 1),
+(2, 1, 'Add title for your Tour.', '', 1, '<p>Here you have to add the title of your Tour Step. </p>', 0, 1, 'bottom', '#jform_title', '', '2021-09-17 18:37:55', 430, '2021-09-17 18:37:55', 430, 1),
+(3, 1, 'Add Content', '', 1, '<p>Add the content of your Tour here!</p>', 0, 1, 'bottom', '#details', '', '2021-09-17 18:39:09', 430, '2021-09-17 18:39:09', 430, 1),
+(4, 1, 'plugin selector', '', 1, '<p>Select the extensions where you want to show your Tour. e.g If you are creating a tour which is only in \'Users\' extensions then select Users here.</p>', 0, 1, 'bottom', '.choices__inner', '', '2021-09-17 18:42:25', 430, '2021-09-17 18:42:25', 430, 1),
+(5, 1, 'URL', '', 1, '<p>Add URL of the page from where you want to start your Tour. </p>', 0, 1, 'bottom', '#jform_url', '','2021-09-17 18:44:26', 430, '2021-09-17 18:48:25', 430, 1),
+(6, 1, 'overlay', '', 1, '<p>Select if you want an overlay effect or not.</p>', 0, 1, 'bottom', '#jform_overlay1', '', '2021-09-17 18:45:34', 430, '2021-09-17 18:48:36', 430, 1),
+(7, 1, 'save and close', '', 1, '', 0, 1, 'bottom', '#save-group-children-save', '', '2021-09-17 18:46:52', 430, '2021-09-17 18:50:35', 430, 1),
+(8, 1, 'Create steps for your Tour', '', 1, '<p>Click on steps icon in the right</p>', 0, 1, 'right', '.btn-info ', '', '2021-09-17 18:52:03', 430, '2021-09-17 18:53:08', 430, 1),
+(9, 1, 'Click here!', '', 1, '<p>Click here to create a new Step for your Tour</p>', 0, 1, 'bottom', '.button-new', '', '2021-09-17 18:55:27', 430, '2021-09-17 18:55:27', 430, 1),
 (10, 1, 'Add title for your Tour.', '', 1, '<p>Here you have to add the title of your Tour Step. </p>', 0, 1, 'bottom', '#jform_title', '', 0, '2021-09-17 18:56:31', 430, '2021-09-17 18:56:31', 430, 1),
```

Comment:
You have to remove the multi_page value here, too. Within one single insert statement you cannot use different columns, you have only one list of columns, and the list of values always have to fit to that.

Reply:
Done the changes